src/cljs/nr/gameboard/board.cljs

```diff
@@ -171,13 +180,15 @@
         ;; Runner clicking on a runner card
         (and (= side :runner)
              (= "Runner" (:side card))
+             (not (any-prompt-open? side))
```

Comment:
Isn't this checked by `playable?`? `:playable` is set by the diffs, which are already checking this, right?

Reply:
So (as far as I can tell) `playable?` is only actually used by the frontend.

*I think* it would be possible to initiate a run, then play a card before the server responds with the state changes that a run is happening, and have the game-state break because of that.

In terms of unit tests, they bypass the `playable` key on the card.

Reply:
cool